docs/src/modules/components/HomeUsers.js

```diff
@@ -51,7 +59,15 @@ const styles = theme => ({
     marginBottom: theme.spacing(5),
   },
   img: {
-    margin: '12px 24px',
+    margin: theme.spacing(1.5, 3),
+    height: 40,
+  },
+  noDescenders: {
+    height: 29,
+  },
+  walmart: {
+    margin: theme.spacing(1.1, 3, 1.5),
```

Comment:
?
```suggestion
    margin: theme.spacing(1, 3, 1.5),
```

Reply:
https://stackoverflow.com/questions/9080633/can-a-css-pixel-be-a-fraction

But `1` is probably close enough if you prefer.